person_class.rb

```diff
@@ -0,0 +1,19 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = nil
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    return false unless of_age? || parent_permission
+  end
```

Comment:
- [ ] Kindly note that this method returns `false` if the value of the `of_age? || parent_permission` condition is false. This means that if the condition resulted in `true`, it will not return anything. Another format you can write this method with is just simply to return the result of the condition:

```ruby
def can_use_services?
   of_age?() || @parent_permission
end
```